src/main/scala/com/github.sstone/amqp/ChannelOwner.scala

```diff
@@ -51,10 +60,9 @@ object ChannelOwner {
           def handleFlow(active: Boolean): Unit = listener ! HandleFlow(active)
         }))
       }
-      case request@Publish(exchange, routingKey, body, properties, mandatory, immediate) => {
+      case request@Publish(_, _, _, _, _, _) => {
```

Comment:
@privateblue @leventem  that `@` vs `:` was strange for me so i looked into it and found that `@` is used when you want an alias for the object, and also match on the values of the case class, so it has nothing to do with what it actually matches. Here it just creates nose, a simple `request: Publish => ` would do the same job in a nicer way.
https://www.artima.com/pins1ed/case-classes-and-pattern-matching.html
```In addition to the standalone variable patterns, you can also add a variable to any other pattern. You simply write the variable name, an at sign (@), and then the pattern. This gives you a variable-binding pattern. The meaning of such a pattern is to perform the pattern match as normal, and if the pattern succeeds, set the variable to the matched object just as with a simple variable pattern.```
http://stackoverflow.com/questions/20748858/pattern-matching-symbol